bool Auto
bool Manual
bool ESTOP
int32 Gear
float64 Speed_mps
float64 Speed_kph
int32 Steer
float64 Steer_degree
int32  Brake
int32 Alive

